sqlmesh/core/state_sync/engine_adapter.py

```diff
@@ -73,9 +74,11 @@ def __init__(
         self,
         engine_adapter: EngineAdapter,
         schema: str = c.SQLMESH,
+        console: t.Optional[Console] = None,
     ):
         self.schema = schema
         self.engine_adapter = engine_adapter
+        self.console = console or get_console()
```

Comment:
TBH, I think if console is None we should just not report.

Reply:
I thought the `console or get_console()` is a common pattern in classes that be instantiated with a `Console` argument. For example, both `Scheduler` and `BuiltInEvaluator` do the same. Fwiw I'm fine with not reporting too.

Reply:
if that's common, feel free to leave this as is